pages/common/npm.md

```diff
@@ -1,10 +1,7 @@
 # npm
 
-> Node package manager, to manage Node.js projects and install module dependencies.
-
-- Create a new project in the current folder:
-
-`npm init`
+> JavaScript and Node.js package manager.
+> Manage Node.js projects and their module dependencies.
 
 - Download all dependencies referenced in package.json:
```

Comment:
I don't think this should be the first example. npm install. It could make sense to come after npm init, given the latter's description.